src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -330,18 +331,11 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
   /**
    * Called when FileUploadPreview tells us about a new or cleared file.
    * This is called right after the File Validity event.
-   * @param file the file to add or null to delete
-   * @param index the index of the file to delete
+   * @param file the file to add
    */
-  async onFileSelected (file: File, index = 0): Promise<void> {
-    if (file) {
-      // verify that file is valid
-      if (!this.fileValidity) {
-        // if file loader hasn't set an error message, set default message
-        if (!this.customErrorMessage) this.customErrorMessage = 'Invalid file.'
```

Comment:
No need to do this -- the file input component will sync any custom errors messages to us and we will display them on line 98.